ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkReader.java

```diff
@@ -213,19 +213,20 @@ private Void readRatioTapChanger(String[] tokens) {
         int num = Integer.parseInt(tokens[1]);
         int tap = Integer.parseInt(tokens[2]);
         String id = mapper.getId(AmplSubset.RATIO_TAP_CHANGER, num);
-        if (id.endsWith(AmplConstants.LEG2_SUFFIX) || id.endsWith(AmplConstants.LEG3_SUFFIX)) {
+        if (id.endsWith(AmplConstants.LEG1_SUFFIX) || id.endsWith(AmplConstants.LEG3_SUFFIX) || id.endsWith(AmplConstants.LEG3_SUFFIX)) {
```

Comment:
2 times LEG3_SUFFIX. I bet the first one is LEG2_SUFFIX, else it is redundant.

Reply:
same remark as @nomont